apps/web-giddh/src/app/services/general.service.ts

```diff
@@ -777,4 +777,14 @@ export class GeneralService {
     public getFileExtension(path: string): string {
         return (path && path.match(/(?:.+..+[^\/]+$)/ig) != null) ? path.split('.').pop() : 'null';
     }
+    /** this will store currency code */
```

Comment:
Do use Document this for adding JS Doc for methods

Reply:
Done